diff -Nur -x '*.orig' -x '*.rej' e-smith-qmailanalog-1.12.0/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/qmailanalog mezzanine_patched_e-smith-qmailanalog-1.12.0/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/qmailanalog
--- e-smith-qmailanalog-1.12.0/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/qmailanalog	2003-06-25 07:07:59.000000000 +1000
+++ mezzanine_patched_e-smith-qmailanalog-1.12.0/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/qmailanalog	2008-02-13 22:57:18.000000000 +1100
@@ -90,12 +90,6 @@
 	<base>END_OF_REPORT</base>
 	<trans>End of Report</trans>
     </entry>
-
-    <entry>
-	<base>Administration</base>
-	<trans>Administration</trans>
-    </entry>
-
     <entry>
 	<base>Mail log file analysis</base>
 	<trans>Mail log file analysis</trans>
